src/coreclr/src/jit/importer.cpp

```diff
@@ -2084,6 +2085,15 @@ GenTree* Compiler::impRuntimeLookupToTree(CORINFO_RESOLVED_TOKEN* pResolvedToken
 
         if (pRuntimeLookup->offsets[i] != 0)
         {
+// The last indirection could be subject to a size check (dynamic dictionary expansion feature)
+#if 0  // Uncomment that block when you add sizeOffset field to pRuntimeLookup.
+            if (i == pRuntimeLookup->indirections - 1 && pRuntimeLookup->sizeOffset != 0xFFFF)
```

Comment:
Should the 0xFFFF be a `#define` in CorInfo.h? (We don't need to add that now, but it might be good to put something like `CORINFO_LOOKUP_DIRECT_OFFSET` here instead of the 0xFFFF so that it forces the addition when `sizeOffset` is added and this block is uncommented.

Reply:
Yep. I can do that with my portion of the changes.